packages/field/src/components/money/index.tsx

```
@@ -0,0 +1,30 @@
+      switch (mode) {
+        case "edit":
+          return (
+            <Input
+              v-model:value={formState![column.value?.dataIndex as string]}
+              placeholder={`请输入${column.value?.title}`}
+              allowClear
+              {...column.value?.fieldProps}
+            />
+          );
+        case "read":
+        default:
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)